### PR TITLE
[#2003] Fix new resource preview error

### PR DESCRIPTION
**Pull request by danielkryska**

### CODE REVIEW

**Comment by Marcelinna** [comment]
changelog

**Comment by Marcelinna** [comment]
Step to reproduce problem
1. Go to backoffice
2. Choose Resource "100 Percent IT Trusted Cloud"
image
3. Click buuton "EDIT"
4. Click button "Preview"
5. See error - resource doesnt have logo
image


beta:
image

**Comment by danielkryska** [comment]
> Step to reproduce problem
> 
> 1. Go to backoffice
> 2. Choose Resource "100 Percent IT Trusted Cloud"
>    image
> 3. Click buuton "EDIT"
> 4. Click button "Preview"
> 5. See error - resource doesnt have logo
>    image
> 
> beta:
> image

Issue fixed

**Comment by Marcelinna** [comment]
**Logo The logo** format you're trying to attach is not supported. Supported formats: png, gif, jpg, jpeg, pjpeg, tiff, vnd.adobe.photoshop or vnd.microsoft.icon.

image


Please delete repeating

**Comment by danielkryska** [comment]
> **Logo The logo** format you're trying to attach is not supported. Supported formats: png, gif, jpg, jpeg, pjpeg, tiff, vnd.adobe.photoshop or vnd.microsoft.icon.
> 
> image
> 
> Please delete repeating

done

File: app/helpers/image_helper.rb [hunk 1]
```diff
@@ -3,6 +3,18 @@
 require "mini_magick"
 
 module ImageHelper
+  @@PERMITTED_EXT_MESSAGE = "^The logo format you're trying to attach is not supported.
+           Supported formats: png, gif, jpg, jpeg, pjpeg, tiff, vnd.adobe.photoshop or vnd.microsoft.icon."
+  @@DEFAULT_LOGO_PATH = Webpacker.manifest.lookup("media/images/eosc-img.png")
+
+  def self.get_default_logo_path
```
Comment (goreck888) [inline app/helpers/image_helper.rb:10]: In rails we avoid to use `get` in the method names.

**Review by goreck888 — CHANGES_REQUESTED** [review]
Just leave `get` in method names. Otherwise everything looks ok

**Comment by Marcelinna** [comment]
Steps to reproduce 
1. Go to /backoffice/services
2. Click the button "Create new resource"
3. Fill required field and add logo in valid format
4. Click button "Preview"
5. Click button "Go back to edit"
6. Click button "Create resource"

See error 
image

**Comment by Marcelinna** [comment]
please add some the tests

**Review by Marcelinna — APPROVED** [review]

**Review by goreck888 — APPROVED** [review]

**Pull request merged**